File: schedules/terraform.ts
// jshvn/terraform -- Cloudflare as code. drift.yml plans every onboarded stack against
// what is live, with no diff filter on purpose, and opens an issue on a finding.
export default {
  repo: "jshvn/terraform",
  workflows: [{ workflow: "drift.yml", slots: ["evening"] }],
} as const

File: schedules/terraform.go
package schedules

// jshvn/terraform is Cloudflare as code. drift.yml plans every onboarded stack against what
// is live, opens an issue on a finding, and pings its own healthcheck.
var _ = register(Job{Repo: "jshvn/terraform", File: "drift.yml", Slots: Evening})
